README.md

```diff
@@ -16,9 +16,20 @@ Run the container
 docker run -it -p 11211:11211 --name memcached modularitycontainers/memcached
 ```
 
-If you would like to increase a CACHE_SIZE use environment variable -e CACHE_SIZE:
+If you would like to debug memcached, use container option =e MEMCACHED_DEBUG_MODE:
 ```bash
-docker run -it -e CACHE_SIZE=128 -p 11211:11211 --name memcached modularitycontainers/memcached
+docker run -it -p 11211:11211
+[-e MEMCACHED_DEBUG_MODE]
+--name memcached modularitycontainers/memcached
+```
+
+If you would like to change memcached options, like cache_size, connections or threads, use environment variable -e MEMCACHED_CACHE_SIZE, -e MEMCACHED_CONNECTIONS, -e MEMCACHED_THREADS respectively:
+```bash
+docker run -it -p 11211:11211
```

Comment:
what am I doing wrong?

```
$ docker run --rm -ti -e MEMCACHED_CACHE_SIZE=111 -e MEMCACHED_CONNECTIONS=25 -e MEMCACHED_THREADS=32 modularitycontainers/memcached
[err] evsig_init: socketpair: Too many open files
```

Reply:
Again missing -p 11211:11211

Reply:
So I would suggest documenting what the defaults are so people are not confused as I was.